pvlib/pvsystem.py

```diff
@@ -2799,7 +2799,7 @@ def scale_voltage_current_power(data, voltage=1, current=1):
     return df_sorted
 
 
-def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
+def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
```

Comment:
```suggestion
from pvlib._deprecation import renamed_kwarg_warning

@renamed_kwarg_warning("0.12.0", "g_poa_effective", "effective_irradiance", "0.13.0")
def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
```
What do you think about deprecating the parameter names with the decorator introduced in #2237?

Reply:
It's obviously a very good idea :)